RedHat-UX/red-hat-design-system
# Updated Theming Overview page and images

## What I did

1. Updated the [Theming Overview page](https://deploy-preview-1938--red-hat-design-system.netlify.app/theming/) with new content and images


## Testing Instructions

1. Check that [the DP](https://deploy-preview-1938--red-hat-design-system.netlify.app/theming/) matches [the design](https://www.figma.com/proto/3liewFtztT3rP4MKCznjVI/Theming?page-id=1%3A88&node-id=1-89&node-type=canvas&viewport=94%2C132%2C1&t=2ITtBYBVgGpBIwcY-1&scaling=scale-down&content-scaling=fixed)

Closes #1931
